src/clients/security.rb

```diff
@@ -180,83 +179,66 @@ def SecuritySummaryHandler(options)
     def SecurityLevelHandler(options)
       options = deep_copy(options)
       current = :custom
-      Builtins.maplist(@Levels) do |key, level|
-        current = key if level == Security.Settings
-      end
-      lvl = ""
-      if options.key?("workstation")
-        lvl = "Level1"
-      elsif options.key?("roaming")
-        lvl = "Level2"
-      elsif options.key?("server")
-        lvl = "Level3"
+
+      levels = @Levels.select { |_key, level| level == Security.Settings }
+
+      current = if levels.empty?
+                  :custom
+                else
+                  levels.last # mimic the old implementation
```

Comment:
this is in fact wrong, as levels contain pair of `[key, level]` and old one use only key.
so how code can look like:
```
current = @Levels.select{ |_key, level| level == Security.Settings }.keys.last || :custom
```